### DISCUSSION TITLE
Simple workaround to easily open 16-bit executables on Windows 11

### DISCUSSION THREAD

**Discussion by zagozago** [otya128/winevdm#1449] [General]
Hello everyone,

First, I would like to thank the devs and everyone contributing to this project! It has been incredibly helpful in running 16-bit apps on 64-bit systems like Win 11.

I'd like to share a simple workaround that I’ve been using to make it easier to open 16-bit executables without needing to manually drag them to otvdm.exe. Here's the solution:

* I renamed my 16-bit executable to a custom extension, like file.e16.
* Then, I attempted to open the file directly on Windows 11, which prompted the system to ask for an app to associate with this unknown file type.
* In the settings, I configured Windows to always open files with the .e16 extension using otvdm.exe.

Now, whenever I double-click a .e16 file, it launches via otvdm.exe automatically, saving me a lot of hassle.

Bonus: This trick also makes it easy to add icons to shortcuts you can create on the Desktop or wherever you like.

Hope this helps someone out there.

**Comment by emendelson** [comment]
But if you run the Install script for otvdm, this shouldn't be necessary. You simply double-click on a 16-bit app and it runs.

**Comment by zagozago** [comment]
As Homer Simpson would say, "DUH!". I hadn't done this process. Thanks for advise!
